pyflagser/flagser.py

```diff
@@ -96,10 +96,11 @@ def flagser_unweighted(adjacency_matrix, min_dimension=0, max_dimension=np.inf,
                                 directed, coeff, _approximation, _filtration)
 
     # Creating dictionary of return values
-    out = dict()
-    out['betti'] = homology[0].get_betti_numbers()
-    out['cell_count'] = homology[0].get_cell_count()
-    out['euler'] = homology[0].get_euler_characteristic()
+    out = {
+        'betti': homology[0].get_betti_numbers(),
```

Comment:
How about putting the `[0]` above so here and below we just call `homology`? Just to avoid repetition (also side note: there are no `[1]` and later entries as far as I can tell, so maybe `compute_homology` should directly return the 0th entry?).

Reply:
I thought I did it, it is corrected now, thanks!